rust/arrow/src/buffer.rs

```diff
@@ -371,118 +388,165 @@ where
 
 fn bitwise_bin_op_helper<F>(
     left: &Buffer,
-    left_offset: usize,
+    left_offset_in_bits: usize,
```

Comment:
💯  for improved naming

Reply:
I wonder too now that you know lots about what `bitwise_bin_op_helper` does, if you might be able to add a summary in the comments

Like
```rust
/// This function creates a new Buffer view aligned on ....
fn bitwise_bin_op_helper<F>(
```